File: app/src/main/java/com/resource/bankapplication/data/remote/LoginService.java
package com.resource.bankapplication.data.remote;

import com.resource.bankapplication.data.remote.dto.UserAccountDto;

import retrofit2.Call;
import retrofit2.http.Field;
import retrofit2.http.FormUrlEncoded;
import retrofit2.http.POST;

public interface LoginService {

    @FormUrlEncoded
    @POST("login")
    Call<UserAccountDto> login(@Field("user") String username, @Field("password") String password);
}

File: app/src/main/java/com/joaoneto/testeandroidv2/loginscreen/service/LoginService.kt
package com.joaoneto.testeandroidv2.loginscreen.service

import com.joaoneto.testeandroidv2.loginscreen.model.LoginResponseModel
import io.reactivex.Observable
import retrofit2.Call
import retrofit2.http.Field
import retrofit2.http.FormUrlEncoded
import retrofit2.http.POST

interface LoginService {
    @FormUrlEncoded
    @POST("login")
    fun signIn(
        @Field("user") user: String,
        @Field("password") password: String
    ): Call<LoginResponseModel>
}